mapstory/templates/maps/map_detail.html

```diff
@@ -29,7 +29,7 @@
   {% autoescape off %}
       $('<iframe>', {
      {% if resource.chapters %}
-     src: '{%  url 'mapstory_view' resource.id %}',
+     src: '{%  url 'mapstory_view' resource.slug %}',
```

Comment:
Also, we'll need to address the share "embed" link generation [here](https://github.com/MapStory/mapstory/blob/master/mapstory/templates/maps/_story_details.html#L81)  as it can't be used now

![screen shot 2017-08-08 at 3 41 39 pm](https://user-images.githubusercontent.com/15912063/29097709-2a65e46c-7c50-11e7-821d-c74b7b00ce57.png)